executor/logger/dispatcher.go

```diff
@@ -1,15 +1,29 @@
 package logger
 
 import (
-	"github.com/go-logr/logr"
 	"os"
+
+	"github.com/go-logr/logr"
+	"github.com/seldonio/seldon-core/executor/api/util"
 )
 
 const (
 	ENV_LOGGER_KAFKA_BROKER = "LOGGER_KAFKA_BROKER"
 	ENV_LOGGER_KAFKA_TOPIC  = "LOGGER_KAFKA_TOPIC"
 )
 
+var WorkerQueue chan chan LogRequest
+
+func getSslElements() *SslKakfa {
```

Comment:
:thought_balloon: Having the struct and accessor function duplicated (here and in `kafka/server.go`) feels a little awkward for maintenance - might be nice to just use one definition for each

Reply:
Yes totally, that's something that I wanted to have access to in https://github.com/SeldonIO/seldon-core/blob/master/executor/cmd/executor/main.go and maybe pass it as an argument to `StartDispatcher()` and `NewKafkaServer()` methods but I couldn't make it possible to access the `struct` defined in loggers/types.go
https://github.com/SeldonIO/seldon-core/pull/3813/files#diff-965116395e4663bd717461c1565c1a0d05c80a9ef69c981c60754c0647df3c9fR27-R33

How would you change the code to make it accessible from `main.go`?

Reply:
So, right now `getSslElements` is _private_ within the `logger` package, but files `cmd` are not a part of that package - they're logically isolated from it so only have access to public members